src/Catalyst.Core.Lib/IO/Handlers/CorrelatableHandler.cs

```diff
@@ -45,7 +45,7 @@ public CorrelatableHandler(T messageCorrelationManager)
         }
         
         /// <inheritdoc />
-        protected override Task WriteAsync0(IChannelHandlerContext context, IMessageDto<ProtocolMessage> message)
+        protected override Task Write0Async(IChannelHandlerContext context, IMessageDto<ProtocolMessage> message)
```

Comment:
Is this to avoid naming warnings? does it still successfully override dotnetty's WriteAsync0?

Reply:
from what I can see WriteAsync0 isn't part of dotnetty it was created as an abstract in OutboundChannelHandlerBase, it might only be there to use generic types instead of unboxing WriteAsync.